libs/common/src/abstractions/organization-user/organization-user.service.ts

```diff
@@ -94,6 +95,20 @@ export abstract class OrganizationUserService {
     ids: string[]
   ): Promise<ListResponse<OrganizationUserBulkResponse>>;
 
+  /**
+   * Accept an invitation to initialize and join an organization created via the Admin Portal **only**.
+   * This only used once for the initial Owner, because it includes the organization's encryption keys.
```

Comment:
```suggestion
   * This is only used once for the initial Owner, because it also creates the organization's encryption keys.
```

Typo in my own suggestion 😁  and a slight tweak while I'm at it.